src/services/ui/src/pages/create/create-options.tsx

```diff
@@ -32,9 +33,10 @@ type OptionsPageProps = {
 };
 /** A page for rendering an array of {@link OptionData} */
 const OptionsPage = ({ options, title, fieldsetLegend }: OptionsPageProps) => {
+  const location = useLocation();
   return (
     <SimplePageContainer>
-      <BreadCrumbs options={BREAD_CRUMB_CONFIG_NEW_SUBMISSION} />
+      <BreadCrumbs options={NEW_SUBMISSION_CRUMBS(location.pathname)} />
```

Comment:
Love this little function here to build out the option. I'm always a big fan of declarative code